src/components/NYPLTextLogo.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const NYPLTextLogo = ({
+  className,
+  width,
+  height,
+  viewBox,
+  style,
+  ariaHidden,
+  title,
+  fill,
+  focusable,
+}) => (
+  <svg
+    className={`${className} svgIcon`}
+    width={width}
+    height={height}
+    viewBox={viewBox}
+    style={style}
+    aria-hidden={ariaHidden}
```

Comment:
Should the fill property be added here? Or should the general CSS color properties for an application set the SVG color?

Reply:
Judging from the rest of the icons in the set, and how the existing logo is being used on the `dgx-react-footer`, I would say the latter is true.